libraries/abstractions/common_io/test/test_iot_pwm.c

```diff
@@ -102,23 +105,24 @@ TEST_GROUP_RUNNER( TEST_IOT_PWM )
     xtestIotPwmGpioSemaphore = xSemaphoreCreateBinary();
     TEST_ASSERT_NOT_EQUAL( NULL, xtestIotPwmGpioSemaphore );
     RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_OpenClose );
-    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_SetGetConfig);
-    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_Start);
-    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_Stop);
-    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_NoConfig);
-    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_OpenFuzzing);
-    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_CloseFuzzing);
-    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_SetConfigFuzzing);
-    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_GetConfigFuzzing);
-    //RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_StartFailure);
-    //RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_OpenFailure );
+    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_SetGetConfig );
+    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_Start );
+    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_Stop );
+    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_NoConfig );
+    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_OpenFuzzing );
+    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_CloseFuzzing );
+    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_SetConfigFuzzing );
+    RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_GetConfigFuzzing );
+    /*RUN_TEST_CASE( TEST_IOT_PWM, AFQP_IotPwm_StartFailure); */
```

Comment:
Do we want to remove these two test cases then?

Reply:
@brianfr I think you made this change. Any comments?